sdk/samples/iot/paho_iot_hub_pnp_component_sample.c

```diff
@@ -391,23 +391,25 @@ static void subscribe_mqtt_client_to_iot_hub_topics(void)
     exit(rc);
   }
 
-  // Messages received on the Twin Patch topic will be updates to the desired properties.
+  // Messages received on the property PATCH topic will be updates to the desired properties.
   rc = MQTTClient_subscribe(
-      mqtt_client, AZ_IOT_PNP_CLIENT_TWIN_PATCH_SUBSCRIBE_TOPIC, IOT_SAMPLE_MQTT_SUBSCRIBE_QOS);
+      mqtt_client, AZ_IOT_PNP_CLIENT_PROPERTY_PATCH_SUBSCRIBE_TOPIC, IOT_SAMPLE_MQTT_SUBSCRIBE_QOS);
   if (rc != MQTTCLIENT_SUCCESS)
   {
     IOT_SAMPLE_LOG_ERROR(
-        "Failed to subscribe to the Twin Patch topic: MQTTClient return code %d.", rc);
+        "Failed to subscribe to the property PATCH topic: MQTTClient return code %d.", rc);
     exit(rc);
   }
 
-  // Messages received on Twin Response topic will be response statuses from the server.
+  // Messages received on property Response topic will be response statuses from the server.
```

Comment:
Nit - should it be `property Response` or `property response`?  Not a bad idea to look at caps after the `Twin` => `property` in general.